<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NodesDialog</class>
 <widget class="QDialog" name="NodesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nodes outputs</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="dispatcherTab">
      <attribute name="title">
       <string>Dispatcher</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTextEdit" name="dispatchertextEdit"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="alarmServerTab">
      <attribute name="title">
       <string>AlarmServer</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTextEdit" name="alarmServerTextEdit"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
